assignments/Action[] actions            # From Recipe
bool in_execution
int32 time_remaining
bool interruptable
